repo/packages/L/linkerd/0/config.json

```diff
@@ -0,0 +1,68 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "required": ["linkerd"],
+  "properties": {
+    "linkerd": {
+      "type": "object",
+      "required": ["admin-port", "config-filename", "config-uri", "cpus", "dns-suffix", "framework-name", "instances", "mem", "name", "routing-port"],
+      "properties": {
+        "admin-port": {
+          "default": 9990,
+          "description": "Admin port. Provides an administrative UI for this instance.",
+          "type": "integer"
+        },
+        "config-filename": {
+          "default": "linkerd-dcos.yaml",
+          "description": "URI of linkerd config file, appends to config-uri. See https://linkerd.io for config docs.",
+          "type": "string"
+        },
+        "config-uri": {
+          "default": "https://s3.amazonaws.com/buoyant-dcos",
+          "description": "URI of linkerd config file, prepends to config-filename.",
+          "type": "string"
+        },
+        "cpus": {
+          "default": 0.25,
+          "description": "CPU shares to allocate to each linkerd instance.",
+          "minimum": 0.25,
+          "type": "number"
+        },
+        "dns-suffix": {
+          "default": ".marathon.mesos",
+          "description": "This value is appended to the framework-name value to form the canonical DNS name for the linkerd service.",
+          "type": "string",
+          "pattern": "^(?:\\.[a-z][a-z0-9]*?(?:-[a-z0-9]+)*)+$"
+        },
+        "framework-name": {
+          "default": "linkerd",
+          "description": "The framework name to register with Mesos.",
+          "type": "string",
+          "pattern": "^/?(([a-z0-9]|[a-z0-9][a-z0-9\\-]*[a-z0-9])\\.)*([a-z0-9]|[a-z0-9][a-z0-9\\-]*[a-z0-9])$"
+        },
+        "instances": {
+          "default": 1,
```

Comment:
we thought about doing this. this leaves linkerd stuck at `Waiting` in the marathon ui. if there's no workaround for that, we may switch this back to `1`, as it makes it look like linkerd is not correctly installed.